pkgs/development/tools/skaffold/default.nix

```diff
@@ -2,8 +2,8 @@
 
 buildGoPackage rec {
   name = "skaffold-${version}";
-  version = "0.30.0";
-  # rev is the 0.30.0 commit, mainly for skaffold version command output
+  version = "0.33.0";
+  # rev is the 0.33.0 commit, mainly for skaffold version command output
   rev = "fe31429012110e6fd70f97971288bd266ba95bed";
```

Comment:
is it normal that the rev wasn't changed even though the comment points at another version ?